doorknob/foyer.py

```diff
@@ -13,10 +13,8 @@ class Scene:
     Aggregate time-series data from objects in a cohesive narrative.
 
     Attributes:
-        aws_data: A list of detected face json objects generated by AWS
-                   Rekognition.
+        aws_data: A list of detected face json objects generated by AWS Rekognition.
```

Comment:
I appreciate that you made the docstring look simpler @uyennguyen16900 . However this line 16 is a little long, it is 85 characters and the Pep 8 standard is to keep it below 80. You don't need to make it go onto the next line, but you please try and make the line length < 80 somehow?